docker-compose.yml

```diff
@@ -410,9 +410,9 @@ services:
       ARROW_MIMALLOC: "ON"
     command: >
       /bin/bash -c "
-        if grep -q -i -e 'centos.* 7' /etc/os-release; then export ARROW_S3=OFF ARROW_MIMALLOC=OFF; fi &&
+        if grep -q -i -e 'centos.* 7' /etc/os-release; then export ARROW_S3=OFF ARROW_GCS=OFF ARROW_MIMALLOC=OFF; fi &&
```

Comment:
On second thought the logic should probably just be removed and the vars set under `environment:`

Reply:
IIUC the intent is that these are only set on centos 7, and you can't conditionally set env vars in `environment:`. Though the setup appears to have baked in centos 7 from where it inherits, so maybe that's not a concern?

Reply:
Yeah this service is fixed to centos 7 so we should be fine using `environment:`